README.md

```diff
@@ -33,9 +34,30 @@ By default, the following loggers are added:
     * logger level can be set by `CONSOLE_LOG_LEVEL` env var (defaults to `silly`).
 
 
-  * The `splunkHEC` logger, if `NODE_ENV === production && SPLUNK_HEC_TOKEN`
+  * The `splunkHEC` logger, if `SPLUNK_HEC_TOKEN` is present
     * logger level can be set by `SPLUNK_LOG_LEVEL` env var (defaults to `warn`).
 
+If you wish to see logs in Splunk that are at levels lower than warn, you can change the default level by setting a `SPLUNK_LOG_LEVEL` environment variable.
+
+### Testing n-logger locally
```

Comment:
Great instructions! I'd suggest they shouldn't be under 'Usage' though, but would probably be good with the same heading after the 'Usage' section.